### PR TITLE
Updated README.md, removed unused variable

### PR DESCRIPTION
**Pull request by devanshjain7**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
* Updated README to add instructions to use multi-cloud with dashboard. 
* Removed an unused variable in ansible installer common.yml file.

### CODE REVIEW

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

File: installer/README.md [hunk 1]
```diff
@@ -29,3 +29,8 @@ ansible-playbook site.yml -i local.hosts
 ```bash
 ansible-playbook clean.yml -i local.hosts
 ```
+### Using Multi-cloud with dashboard
+* Install dashboard by following the dashboard installation guide [here](https://github.com/anvithks/opensds-dashboard/blob/dashboard-ansible-installer/install/readme.md).
```
Comment (joseph-v) [inline installer/README.md:33]: Can you please remove/update personal link for documentation

Comment (devanshjain7) [reply installer/README.md]: done.

**Review by joseph-v — APPROVED** [review]
LGTM

**Pull request merged**